packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -4,123 +4,134 @@ section: components
+Toast alerts are commonly used in the following situations:
+- Confirming that a message sent
+- Confirming that an update was made
+- Informing the user that their action was completely successfully
+- Informing the user that their action was completed with errors
 
-### Toast alerts
-Toast alerts enter from the right edge of the screen and remain until they time out or until the user dismisses them. Toast alerts overlay other content on the page.
+## Content
 
-![toast notification placement](./img/toast-notification.png)
 
-See a [high-fidelity example of toast alert placement](#toast-alert-placement) below.
+- In inline alerts and toast alerts, avoid using “success” and “successfully.” They’re extraneous.
 
-#### When to use
-Use toast alerts when you want to report an asynchronous event without interrupting the user’s current workflow. Actions and links may be used in alerts, but only as a shortcut or redundant means to navigate or invoke the action. A toast alert should never be used as the only means for taking actions on an alert event.
+<div class="ws-content-table">
 
-Toast notifications must always have a close button so the user can dismiss them. You may also include an optional time-out to dismiss the alert automatically after a period of time. Eight (8) seconds is a recommended timeout. Alerts should not dismiss while the user is hovering over them.
+| **Do** | **Don’t** |
+|--------|-----------|
+| Account created| Your account was successfully created.|
 
-#### When not to use
-Do not use a toast alert when an event requires immediate action or for synchronous events like form validation.
+</div>
 
-### Stacking multiple alerts
-Multiple alerts will stack. The most recent alert should appear at the top and others will push down. When alerts are deleted, any space above will be filled by pushing remaining alerts up in the stack order.
+<br /> 
 
-An [alert group](/components/alert-group) component allows a set of alerts to be properly positioned and stacked as shown below.
+- Be concise. In one-line toast alerts, use fragments instead of sentences if possible.
 
-![alert group](./img/toast-notification-multiple.png)
+<div class="ws-content-table">
 
-### Examples
+| **Do** | **Don’t** |
+|--------|-----------|
+| Message sent| Your message has been sent.|
 
-#### Toast alert placement
-Toast alerts always stack in the upper right of a page.
+</div> 
 
-<img src="./img/Toast-alerts-desktop.png" alt="toast alerts in a page" />
+<br />
 
+- In toast alerts containing links, describe how the user can navigate through the UI to perform the action. Learn more about how descriptive text supports accessibility in the next section.
 
-#### Inline alert placement
-If an alert message applies globally to the content on a page, place the alert in the page header area just below the title.
+| **Do** | **Don’t** |
```

Comment:
@redallen this table's column distribution differs from the rest - does that have anything to do with the recent PR merge for table columns?

Reply:
No, it just needs to be wrapped in `<div class="ws-content-table">` like the other.

Reply:
Thanks, Zack!

Reply:
Do you mind adding those wrappings Abi? It just looks a little weird right now how different the table proportions are

Reply:
I added those wrappings, @mmenestr ! They were included in my most recent commit this morning

Reply:
Oh I don't see them in the PR? Unless i'm not looking in the right place. But it doesn't seem like the changes are reflected in the preview either hmmm